src/ui/controllers/xapi.ts

```diff
@@ -12,7 +13,7 @@ export async function proxy(ireq: express.Request, res: express.Response) {
 
 	const agent = {
 		account: {
-			homePage: 'https://cslearning.gov.uk',
+			homePage: xapi.HomePage,
```

Comment:
Nice catch!

Reply:
Wouldn't have noticed but the missing trailing slash meant xapi didn't return results matching the user for these statements 🙄